Resources/views/includes/bootstrap.html.twig

```diff
@@ -4,7 +4,7 @@
     <link href="{{ asset_url }}" type="text/css" rel="stylesheet"/>
 {% endstylesheets %}
 
-<div id="carousel-{{ widget.id }}" class="carousel slide" data-ride="carousel" data-interval="0">
+<div id="carousel-{{ widget.id }}" class="carousel slide" data-ride="carousel" data-interval="0" style="min-height: 200px;">
```

Comment:
I'd rather you adapt the style of the edit button instead of doing this trick... think about a tiny slider.. it won't work then